modules/swagger-codegen/src/main/resources/swift3/Models.mustache

```diff
@@ -10,8 +10,10 @@ protocol JSONEncodable {
     func encodeToJSON() -> Any
 }
 
-public enum ErrorResponse : Error {
-    case Error(Int, Data?, Error)
+public struct ErrorResponse : Error {
+    public let statusCode: Int
+    public let data: Data?
+    public let error: Error
 }
```

Comment:
Even without making this `CustomStringConvertible`, Swift's default string conversion for structs will still let you see all the data - if you have a suggestion for a better format though, I'd be happy to implement it.